resources/qml/Menus/ProfileMenu.qml

```diff
@@ -17,7 +17,11 @@ Menu
 
         MenuItem
         {
-            text: (model.layer_height != "") ? model.name + " - " + model.layer_height + model.layer_height_unit : model.name
+            text: {
```

Comment:
Some minor codestyle issues. Fixed those in d09d8a6ae34f75b3a7a3d6bc0a20add4671c7e9c